core/src/foundations/ionic.vars.scss

```diff
@@ -1067,7 +1093,7 @@ $ion-heading-h6-bold: (
   text-decoration: none,
 );
 
-$ion-body-action-xl: (
+$ion-body-action-lg: (
```

Comment:
Will the Figma designs be updated to use these new names?

Reply:
My guess is not since as of last week, the design team has shifted to a new board so I would assume that they won't be updating the old one.

Reply:
I talked with UX/UI and yes, they intend to be updating the Dear Developer pages, but its an ongoing process still :)